catalogue/webapp/utils/json-ld.ts

```diff
@@ -0,0 +1,33 @@
+import { Work as WorkType } from '@weco/common/model/catalogue';
+import { objToJsonLd } from '@weco/common/utils/json-ld';
+
+export function workLd(work: WorkType) {
+  const creators = (work.contributors || []).map(c => {
+    return {
+      '@type': c.agent.type,
+      name: c.agent.label,
+    };
+  });
+
+  const keywords = work.subjects.map(s => s.label).join(',');
+
+  return objToJsonLd(
+    {
+      additionalType: null, // TODO: needs API
+      locationCreated: null, // TODO: needs API
+      genre: null, // TODO: needs API
+      datePublished: null, // TODO: needs API
+      dateCreated: null, // TODO: needs API
+      dateModified: null, // TODO: needs API
+      alternativeHeadline: null, // TODO: needs API
+      publishedBy: null, // TODO: needs API
```

Comment:
We have been trying to get rid of `null`s unless they're absolutely necessary - can you make these `undefined`s?